_posts/2025-02-04-introducing-reciprocal-rank-fusion-hybrid-search.md

```diff
@@ -0,0 +1,345 @@
+| FIQA	      | 0.2389	  | 0.2004	 | 0.2693  | 	0.2474         | 	8.13%             |
+| Trec-Covid | 	0.6087	 | 0.2718  | 	0.5905 | 	0.5877         | 	0.47%             |
+| SciDocs    | 	0.155   | 	0.1075 | 	0.1602 | 	0.1525         | 	4.81%             |
+| Quora	     | 0.7424	  | 0.8256	 | 0.8452	 | 0.796	          | 5.82%              |
+|            |          |         |         | **Average:**	   | 3.86%              |
+
+### Search latency 
+
+The following table presents search latency measurements in milliseconds at different percentiles (p50, p90, and p99) for both Hybrid and RRF approaches. The percent difference columns show the relative performance impact between these methods.
+
+<table>
+  <tr>
+    <th></th>
+    <th colspan="3"><b>p50</b></th>
+    <th colspan="3"><b>p90</b></th>
+    <th colspan="3"><b>p99</b></th>
+  </tr>
+  <tr>
+    <td></td>
+    <td><b>Hybrid</b></td>
+    <td><b>RRF</b></td>
+    <td><b>Percent difference</b></td>
+    <td><b>Hybrid</b></td>
+    <td><b>RRF</b></td>
+    <td><b>Percent difference</b></td>
+    <td><b>Hybrid</b></td>
+    <td><b>RRF</b></td>
+    <td><b>Percent difference</b></td>
+  </tr>
+  <tr>
+    <td>NFCorpus</td>
+    <td>71</td>
+    <td>67.5</td>
+    <td>4.93%</td>
+    <td>95</td>
+    <td>89.9</td>
+    <td>5.37%</td>
+    <td>112.84</td>
+    <td>109.56</td>
+    <td>2.91%</td>
+  </tr>
+  <tr>
+    <td>ArguAna</td>
+    <td>390.5</td>
+    <td>390</td>
+    <td>0.13%</td>
+    <td>423.5</td>
+    <td>424</td>
+    <td>-0.12%</td>
+    <td>459.475</td>
+    <td>458.5</td>
+    <td>0.21%</td>
+  </tr>
+  <tr>
+    <td>FIQA</td>
+    <td>109</td>
+    <td>105.25</td>
+    <td>3.44%</td>
+    <td>139.65</td>
+    <td>137.5</td>
+    <td>1.54%</td>
+    <td>165.76</td>
+    <td>160.79</td>
+    <td>3.00%</td>
+  </tr>
+  <tr>
+    <td>Trec-Covid</td>
+    <td>165.5</td>
+    <td>159.75</td>
+    <td>3.47%</td>
+    <td>209.5</td>
+    <td>203.6</td>
+    <td>2.82%</td>
+    <td>240.73</td>
+    <td>237.78</td>
+    <td>1.23%</td>
+  </tr>
+  <tr>
+    <td>SciDocs</td>
+    <td>103</td>
+    <td>103.5</td>
+    <td>-0.49%</td>
+    <td>126.5</td>
+    <td>126.05</td>
+    <td>0.36%</td>
+    <td>156.02</td>
+    <td>159.015</td>
+    <td>-1.92%</td>
+  </tr>
+  <tr>
+    <td>Quora</td>
+    <td>167</td>
+    <td>170</td>
+    <td>-1.80%</td>
+    <td>209.5</td>
+    <td>212.55</td>
+    <td>-1.46%</td>
+    <td>264.5</td>
+    <td>266.51</td>
+    <td>-0.76%</td>
+  </tr>
+  <tr>
+    <td></td>
+    <td></td>
+    <td><b>Average:</b></td>
+    <td>1.62%</td>
+    <td></td>
+    <td><b>Average:</b></td>
+    <td>1.42%</td>
+    <td></td>
+    <td><b>Average:</b></td>
+    <td>0.78%</td>
+  </tr>
+</table>
+
+### CPU utilization
+
+The following table shows CPU utilization measurements on the coordinator node, comparing hybrid and RRF approaches. The percent difference column indicates the relative change in CPU usage between the two methods.
+
```

Comment:
I went with italics for the name of the column, but you can choose different formatting if you'd like.